src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -99,91 +79,12 @@ namespace gauges {
  *   - Exponents \f$ e_X\f$ are taken as input here as `exp_X`.
  *   - Spatial weight function \f$W\f$ is specified completely by
  *     \f$\sigma_r\f$, which is taken as input here as `sigma_r`.
- */
-template <size_t SpatialDim, typename Frame>
-void damped_harmonic_h(
-    gsl::not_null<db::item_type<Tags::GaugeH<SpatialDim, Frame>>*> gauge_h,
-    const db::const_item_type<Tags::InitialGaugeH<SpatialDim, Frame>>&
-        gauge_h_init,
-    const Scalar<DataVector>& lapse,
-    const tnsr::I<DataVector, SpatialDim, Frame>& shift,
-    const Scalar<DataVector>& sqrt_det_spatial_metric,
-    const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
-    double time, const tnsr::I<DataVector, SpatialDim, Frame>& coords,
-    // Scaling coeffs in front of each term
-    double amp_coef_L1, double amp_coef_L2, double amp_coef_S,
-    // exponents
-    int exp_L1, int exp_L2, int exp_S,
-    // roll on function parameters for lapse / shift terms
-    double t_start_h_init, double sigma_t_h_init, double t_start_L1,
-    double sigma_t_L1, double t_start_L2, double sigma_t_L2, double t_start_S,
-    double sigma_t_S,
-    // weight function
-    double sigma_r) noexcept;
-
-/*!
- * \brief Damped harmonic gauge source function.
  *
- * \details See damped_harmonic_h() for details.
+ * Compute spacetime derivatives, i.e. \f$\partial_a H_b\f$, of the damped
```

Comment:
"Computes" for consistency with the above text, or "Also computes"?

Reply:
Done in #2292